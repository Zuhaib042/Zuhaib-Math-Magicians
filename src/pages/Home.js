import React from 'react';
import Header from '../components/Header';

export default function Home() {
  return (
    <div>
      <Header />
      <h1 className="text-4xl text-white font-semibold my-12 px-2 sm:ml-6">
        Welcome to our Page!
      </h1>
      <p className="text-white leading-5 px-2 sm:mx-6">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad, illo.
        Aspernatur amet nesciunt maxime esse ratione, voluptatibus molestiae
        iusto assumenda dolores sed labore, eligendi eaque earum reprehenderit,
        pariatur iste incidunt. Lorem ipsum dolor sit amet consectetur
        adipisicing elit. Ad, illo. Aspernatur amet nesciunt maxime esse
        ratione, voluptatibus molestiae iusto assumenda dolores sed labore,
        eligendi eaque earum reprehenderit, pariatur iste incidunt. Lorem ipsum
        dolor sit amet consectetur adipisicing elit. Ad, illo. Aspernatur amet
        nesciunt maxime esse ratione, voluptatibus molestiae iusto assumenda
        dolores sed labore, eligendi eaque earum reprehenderit, pariatur iste
        incidunt. Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad,
        illo.
      </p>
      <p className="text-white leading-5 px-2 sm:mx-6">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad, illo.
        Aspernatur amet nesciunt maxime esse ratione, voluptatibus molestiae
        iusto assumenda dolores sed labore, eligendi eaque earum reprehenderit,
        pariatur iste incidunt. Lorem ipsum dolor sit amet consectetur
        adipisicing elit. Ad, illo. Aspernatur amet nesciunt maxime esse
        ratione, voluptatibus molestiae iusto assumenda dolores sed labore,
        eligendi eaque earum reprehenderit, pariatur iste incidunt. Lorem ipsum
        dolor sit amet consectetur adipisicing elit. Ad, illo. Aspernatur amet
        nesciunt maxime esse ratione, voluptatibus molestiae iusto assumenda
        dolores sed labore, eligendi eaque earum reprehenderit, pariatur iste
        incidunt. Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad,
        illo.
      </p>
    </div>
  );
}
