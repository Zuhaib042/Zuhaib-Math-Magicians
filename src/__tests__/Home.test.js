import { render, screen } from '@testing-library/react';
import Home from '../pages/Home';

describe('Homepage', () => {
  it('Check if Home page rendered properly', () => {
    render(<Home />);
    const home = screen.getByText(/welcome to our page/i);

    expect(home).toMatchSnapshot();
  });

  it('Check if Home page rendered properly', () => {
    render(<Home />);
    const home = screen.getByRole('heading', { name: /Welcome to our page/i });

    expect(home).toMatchSnapshot();
  });
});
