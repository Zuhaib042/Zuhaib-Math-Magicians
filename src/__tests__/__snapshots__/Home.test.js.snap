// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage Check if Home page rendered properly 1`] = `
<h1
  class="text-4xl text-white font-semibold my-12 px-2 sm:ml-6"
>
  Welcome to our Page!
</h1>
`;

exports[`Homepage Check if Home page rendered properly 2`] = `
<h1
  class="text-4xl text-white font-semibold my-12 px-2 sm:ml-6"
>
  Welcome to our Page!
</h1>
`;
